sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/TagService.java

```diff
@@ -0,0 +1,7 @@
+package com.exadel.sandbox.team5.service;
+
+import com.exadel.sandbox.team5.entity.Tag;
+
+public interface TagService extends CRUDService<Tag>{
```

Comment:
It is also redundant for now. Tags don't exist without selected business object.